### PR TITLE
Test to udp_server

### PR DESCRIPTION
**Pull request by xkevin190**
### Contribution description

tests were added to the udp server module, in these tests emUnit was not used due to errors when executing the callback within the emUnit thread, to do the udp server tests the server was started and a python script was added where it would verify correct data reception.

this PULL REQUEST is related to PR #192 of client udp because some necessary modifications were made that are in that PR, PR #192 must be mixed first.

### Testing procedure

cd test/udp_server

make BOARD="your_board" flash test

### CODE REVIEW

File: tests/udp_server/main.c [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @brief       storage file
+ *
```
Comment (eduazocar) [inline tests/udp_server/main.c:19]: Refer this a test_file of udp_server

File: tests/udp_server/main.c [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+#include "shell.h"
+#include "shell_commands.h"
+#include "msg.h"
+#include "xtimer.h"
+#include "board.h"
+
+int port = 3000;
+
+void udp_callback(uint8_t *message, size_t len) {
+    (void)len;
```
Comment (eduazocar) [inline tests/udp_server/main.c:39]: Why cast the len params?

**Review by eduazocar — COMMENTED** [review]
Do Squash to aprrove this

File: tests/udp_server/main.c [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+ */
+#include <string.h>
+#include <errno.h>
+#include <stdio.h>
+
+#include "embUnit.h"
+#include "udp_client.h"
+#include "udp_server.h"
+
+#include "shell.h"
```
Comment (RocioRojas) [inline tests/udp_server/main.c:30]: remove these includes

**Review by eduazocar — COMMENTED** [review]
Udp_server looks good man, Congratulations, Do squash to merge it soon :+1:

File: tests/udp_server/main.c [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+#include "shell_commands.h"
```
Comment (eduazocar) [inline tests/udp_server/main.c:31]: These Libraries are used?

**Review by eduazocar — APPROVED** [review]
Approved :+1:

**Pull request merged**